File: Programmers/Lv.1 - K번째수/K번째수.js
function solution(array, commands) {
    var answer = [];
    for(var i = 0; i < commands.length; i++) {
        var temp = [];
        for(var j = commands[i][0]-1; j < commands[i][1]; j++) {
            temp.push(array[j]);
        }
        temp.sort((a, b) => a - b);
        answer.push(temp[commands[i][2]-1]);
    }
    return answer;
}

File: Programmers/Lv.1 - K번째수/K번째수.py
def solution(array, commands):
    answer = []
    for i in range(len(commands)):
        temp = []
        for j in range(commands[i][0]-1, commands[i][1]):
            temp.append(array[j])
        temp.sort()
        answer.append(temp[commands[i][2]-1])
    return answer